static/templates/presence_rows.hbs

```diff
@@ -0,0 +1,3 @@
+{{#each row_members}}
```

Comment:
Why the name `row_members` as opposed to something like `presence_rows`?

Reply:
fixed!